plugins/ldap/server/__init__.py

```diff
@@ -145,8 +145,11 @@ def _ldapAuth(event):
             conn.bind_s(server['bindName'], server['password'], ldap.AUTH_SIMPLE)
 
             searchStr = '%s=%s' % (server['searchField'], login)
-            results = conn.search_s(server['baseDn'], ldap.SCOPE_ONELEVEL, searchStr, _LDAP_ATTRS)
-
+            # Add the searchStr to the attributes, keep local scope.
+            temp_attr = [server['searchField']]
+            temp_attr.extend(_LDAP_ATTRS)
+            lattr = tuple(temp_attr)
```

Comment:
@nicholsn  Thanks for the fix. We can combine the above three lines into just:

`lattr = _LDAP_ATTRS + (server['searchField'],)`

Reply:
👍 good point.... I'll push up a fix shortly